MAUI/DataGrid/Grouping.md

```diff
@@ -88,6 +88,57 @@ this.dataGrid.IndentColumnWidth = 60;
 {% endhighlight %}
 {% endtabs %}
 
+### Customize indent column background color
+
+Set the background colors to indent cells based on the column where indent cells are present. Refer to the following code snippet to apply background color to indent cells based on the column index.
```

Comment:
The indent cells can be customized by the writing style for [DataGridIntentCell]() TargetType.

The following code snippet shows how to apply a background color to the indent cell based on the row type:

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/32b4ea1b-03d7-463c-9fe0-b36b531c21eb)